app/coffeescript/components/ui/clusters.coffee

```diff
@@ -5,25 +5,23 @@ define [
   'flight/lib/component',
   'marker-clusterer',
   'map/components/utils/map_utils',
-  'map/components/utils/mobile_detection'
+  'map/components/utils/mobile_detection',
+  'map/components/utils/cluster_opts'
 ], (
   compose,
   defineComponent,
   markerClusterer,
   map_utils,
-  mobileDetection
+  mobileDetection,
+  clusterOpts
 ) ->
 
   initMarkerClusters = ->
```

Comment:
i might be a little foggy on the stories AC - is goal to be able to pass in custom options to market cluster?